src/bitcoin-cli.cpp

```diff
@@ -151,10 +151,15 @@ static int AppInitRPC(int argc, char* argv[])
             strUsage += FormatParagraph(LicenseInfo());
         } else {
             strUsage += "\n"
-                "Usage:  bitcoin-cli [options] <command> [params]  Send command to " PACKAGE_NAME "\n"
-                "or:     bitcoin-cli [options] -named <command> [name=value]...  Send command to " PACKAGE_NAME " (with named arguments)\n"
-                "or:     bitcoin-cli [options] help                List commands\n"
-                "or:     bitcoin-cli [options] help <command>      Get help for a command\n";
+                "The bitcoin-cli utility provides a command line interface to interact with a bitcoind/bitcoin-qt RPC server.\n"
+                "\nIt can be used to query network information, manage wallets, create or broadcast transactions, and control the bitcoind server.\n"
+                "\nUse the \"help\" command to list all commands. Use \"help <command>\" to show help for that command.\n"
+                "Using the -named option allows specifying params using key=value style, and therefore doesn't require passing unused params.\n"
+                "\nUsage: bitcoin-cli [options] <command> [params]\n"
+                "or: bitcoin-cli [options] -named <command> [name=value]...\n"
```

Comment:
Should we be keeping the spaces to line them up?

Reply:
Re-added the spacing